client/scss/layouts/_login.scss

```diff
@@ -23,32 +33,60 @@ $desktop-nice-padding: 100px;
   }
 
   h1 {
-    font-weight: 300;
-    font-size: 2em;
-    line-height: 1em;
-    color: $color-white;
+    color: theme('colors.primary.DEFAULT');
     white-space: nowrap;
+    font-weight: 800;
+    font-size: 30px;
+    line-height: 130%;
```

Comment:
Avoid line height if you can. It creates some complexity in the styles. 
Also maybe this would suit using `@apply` - for the h1 styles. See other usage for example